README.md

```diff
@@ -10,6 +10,22 @@ It is implemented as a Firefox Add-on, using the [Add-on SDK](https://developer.
 
 Ian has been blogging about the [design, definition, and development process](http://www.ianbicking.org/tag/product-journal.html).
 
+### Installation and Setup
+
+Install [Postgres](http://www.postgresql.org/).
+
+Install [cfx](https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/cfx).
+
+Install [node.js](https://nodejs.org/). Version 0.12 is required.
+
+Clone the [repository](https://github.com/mozilla-services/pageshot/).
+
+Inside the local clone, run `npm install`.
+
+Inside the clone, run `gulp`. `gulp` will run the server and display the output in this terminal. It will also create the addon and run a copy of Firefox with the addon installed. To see the output from this Firefox process, run `tail -f addon.log` in another terminal window.
```

Comment:
I think either you have to do `npm install -g gulp` or else do `node_modules/.bin/gulp` ?